packages/components/src/utils/colors-values.js

```diff
@@ -1,181 +1,86 @@
-/**
- * External dependencies
- */
-import { merge } from 'lodash';
-
 /**
  * Internal dependencies
  */
 import { rgba } from './colors';
 
-const BASE = {
-	black: '#000',
-	white: '#fff',
-};
+const white = '#fff';
 
-const G2 = {
-	blue: {
-		medium: {
-			focus: '#007cba',
-			focusDark: '#fff',
-		},
-	},
-	gray: {
-		900: '#1e1e1e',
-		700: '#757575', // Meets 4.6:1 text contrast against white.
-		600: '#949494', // Meets 3:1 UI or large text contrast against white.
-		400: '#ccc',
-		300: '#ddd', // Used for most borders.
-		200: '#e0e0e0', // Used sparingly for light borders.
-		100: '#f0f0f0', // Used for light gray backgrounds.
-	},
-	mediumGray: {
-		text: '#757575',
-	},
-	lightGray: {
-		ui: '#949494',
-		secondary: '#ccc',
-		tertiary: '#e7e8e9',
-	},
+// Matches the grays in @wordpress/base-styles
```

Comment:
It would great if we could, in the future, use the same values from `@wordpress/base-styles` without having to manually re-state them.

We could either:

- declare those values in `@wordpress/base-styles` as CSS variables, OR
- in the context of allowing some theming to `@wordpress/components`, expose some CSS variables and therefore allow consumers of the components package to override these values.

The second option would be my favourite, as it decouple the package better. But I also recognize that we can't expose every single value as a CSS variable, as it can quickly grow to become overwhelming.